File: JAVA/SequenciaImpares.java
import java.util.Scanner;

public class SequenciaImpares {
    public static void main(String[]args) {
        Scanner s = new Scanner(System.in);
        System.out.print("Digite o valor de X: ");
        int x = Integer.parseInt(s.next());
        for(int i = 1; i <=x; i+=2){
            System.out.println(i);
        }
        s.close();
    }
}

File: PYTHON/sequencia_impares.py
class SequenciaImpares:
    def exibirNumeros(x):
                  # range(star,stop,step)
        for num in range(1,x,2):
            print(num)


if __name__=='__main__':
    x = int(input("Digite o valor de x: "))
    SequenciaImpares.exibirNumeros(x)